tests/integration/mixer/outboundtrafficpolicy/traffic_registry_only_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package outboundtrafficpolicy
+
+import (
+	"testing"
+)
+
+func TestOutboundTrafficPolicy_RegistryOnly(t *testing.T) {
+	expected := map[string]Response{
+		"http": {
+			Metric:    "istio_requests_total",
+			PromQuery: `sum(istio_requests_total{destination_service_name="BlackHoleCluster",response_code="502"})`,
+			Code:      []string{"502"},
+		}, // HTTP will return an error code
+		"http_egress": {
+			Metric:    "istio_requests_total",
+			PromQuery: `sum(istio_requests_total{destination_service_name="istio-egressgateway",response_code="200"})`,
```

Comment:
`destination_service_name ` for `http_egress` test case in both registry only and allow any should be `istio-egressgateway`